ispdb/fastwebnet.it.xml

```diff
@@ -1,12 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
 <clientConfig version="1.1">
   <emailProvider id="fastwebnet.it">
     <domain>fastwebnet.it</domain>
-    <displayName>fastwebnet.it</displayName>
-    <displayShortName>fastwebnet.it</displayShortName>
+    <displayName>FASTMail</displayName>
```

Comment:
This probably should be Fastweb as well, especially since another big email provider we support is also named Fastmail (https://fastmail.com) and using the same name here would likely cause some confusion.

```suggestion
    <displayName>Fastweb</displayName>
```